### DISCUSSION TITLE
Callipeg Update (Out of Pegs Needs to be on ToonSquid ASAP)

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#388] [General]
Guys, Callipeg just added Out of Pegs. I'm not trying to promote it, but this post is to inform Keiwan that the Out of Pegs feature needs to be implemented as soon as possible, because I don't want people having to pay $50 for a feature that could be accessible for $10, like I did when I bought Callipeg for it having line smoothing.

**Comment by mshoek** [comment]
For me, it's not just about the price. My biggest issue with Callipeg is that it creates ridiculously huge files and crashes all the time, even though I have an M1 chip iPad Pro. I've just made 10 small, rough drawings in a full HD project to try out the Out of Pegs feature, and it's already around 100 MB... It's ridiculous. I can't even imagine using it on regular iPads, so I usually advise against buying it. The Out of Pegs feature works quite well, though. I really, really hope it will be implemented in ToonSquid soon!

Right now, the only things I like better in Callipeg are the line smoothing (it just feels more natural), the floating menu, and this new Out of Pegs feature. In my opinion, ToonSquid is superior in everything else.
